### ISSUE TITLE
[FEATURE] Set up N user-mode VMs at Gamma and Johnson

### ISSUE THREAD

**Issue by FatmanUK** [FatmanUK/dreamtrack23#24]

**Comment by FatmanUK** [comment]
Blocked on #23 .

**Comment by FatmanUK** [comment]
Complete for Johnson. Gamma still dead.

**Comment by FatmanUK** [comment]
Deferring this as blocking ticket is deferred.
